LINUX/i40e_netmap_linux.h

```diff
@@ -470,7 +471,13 @@ i40e_netmap_rxsync(struct netmap_kring *kring, int flags)
 			}
 			ring->slot[nm_i].len = ((qword & I40E_RXD_QW1_LENGTH_PBUF_MASK)
 			    >> I40E_RXD_QW1_LENGTH_PBUF_SHIFT) - crclen;
-			ring->slot[nm_i].flags = slot_flags;
+
+			curr_slot_flag = slot_flags;
+			if (unlikely((staterr & (1<<I40E_RX_DESC_STATUS_EOF_SHIFT)) == 0 )) {
+				curr_slot_flag |= NS_MOREFRAG;
```

Comment:
of course the compiler would optimize it: my point is only about readability, since the new variable is distracting. Anyway, this is not really important. We can merge the the patch as is, if you like.